hathor/conf/settings.py

```diff
@@ -373,3 +373,9 @@ def MAXIMUM_NUMBER_OF_HALVINGS(self) -> int:
     EVENT_API_DEFAULT_BATCH_SIZE: int = 100
 
     EVENT_API_MAX_BATCH_SIZE: int = 1000
+
+    # Maximum number of sync running simultaneously.
+    MAX_ENABLED_SYNC: int = 8
+
+    # Peers to always enable sync.
+    ALWAYS_ENABLE_SYNC: Set[str] = set()
```

Comment:
Are we going to set peer IDs here? Shouldn't we use an approach like the whitelist, downloading it from an URL?

Reply:
You're right. Let me change this part.

Reply:
Done! Thanks!